bedrock/firefox/tests/test_base.py

```diff
@@ -1121,22 +1121,17 @@ def setUp(self):
                       'Gecko/20100101 Firefox/35.0')
         self.url = reverse('firefox.hello.start', args=['35.0'])
 
-    def test_fx_hello_no_conversation(self):
-        """Should identify when there is no conversation"""
+    def test_fx_hello_redirect_non_firefox(self):
+        """Should redirect to /firefox/new if not on Firefox"""
 
-        response = self.client.get(self.url, HTTP_USER_AGENT=self.user_agent)
-        self.assertIn('data-incoming-conversation="none"', response.content)
-
-    def test_fx_hello_conversation_open(self):
-        """Should identify when a conversation is open"""
-
-        response = self.client.get(self.url + '?incomingConversation=open',
-            HTTP_USER_AGENT=self.user_agent)
-        self.assertIn('data-incoming-conversation="open"', response.content)
+        self.user_agent = ('Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36')
+        self.url = reverse('firefox.hello.start', args=['35.0'])
+        response = self.client.get(self.url)
+        self.assertIn('/firefox/new/', response['location'])
 
-    def test_fx_hello_conversation_waiting(self):
-        """Should identify when a conversation is waiting"""
+    def test_fx_hello_redirect_not_updated(self):
```

Comment:
I'm pretty sure this test is giving a false positive. Afaik, there is no code in place to redirect Firefox versions < 35 to the `/new` page. This page is only accessible via an in-product link that was added in 35. If someone does share the URL with a user running an older Firefox version, they will just get the regular default page and no tour.

If we want to put a redirect in place, I suggest we do this via a new bug?